project/nextjs.py

```diff
@@ -0,0 +1,6 @@
+from . import BaseProject
+
+class Nextjs(BaseProject):
+    updateCommands = {
+        'yarn.lock': 'yarn upgrade'
+    }
```

Comment:
Why not just add this to the main list in the parent class?  There's nothing Nextjs specific about it.

Reply:
Because package-lock won't ever be used, but it will be created when going through the whole list of `updateCommands` - so it's just an extra file in the repo with no use, pure template-builder artifact. Nextjs recommends using yarn, and running `yarn` with a `yarn.lock` and a `package-lock.json` gives you a warning that it's not recommended to have both lock files present.

Reply:
So unless we introduce some logic into `update` & `package_update_actions()` to only `yarn upgrade` if there's a `yarn.lock` (if present, then skip `npm update`) this seemed like the quickest fix.